code/extensions/CommentsExtension.php

```diff
@@ -423,7 +423,18 @@ public function CommentsForm() {
 			Requirements::javascript(THIRDPARTY_DIR . '/jquery-validate/lib/jquery.form.js');
 			Requirements::javascript(COMMENTS_THIRDPARTY . '/jquery-validate/jquery.validate.min.js');
 			Requirements::javascript('comments/javascript/CommentsInterface.js');
```

Comment:
We can't do the language files, since they could be per-request dependant.

Maybe everything except jquery + entwine?